back-end/api/quickstart/tests.py

```diff
@@ -31,6 +32,36 @@ def test_get_invalid_author(self):
     self.assertEqual(response.status_code, status.HTTP_404_NOT_FOUND)
 
 
+class UpdateAuthorById(TestCase):
+  """Tests for updating a single Author by PUT'ing to endpoint /author/_id/.
+  Referenced https://realpython.com/test-driven-development-of-a-django-restful-api/
+  """
+  def setUp(self):
+    self.john = Author.objects.create(
+      _id='testId', 
+      displayName='John Doe',
+      url="testUrl",
+      github="testGithub"
+    )
+
+    self.payload = {
+      # TODO: Remove _id here once we figure out the _id field. It should be uneditable.
+      '_id': 'testId',
+      'displayName': 'new John Doe',
+      'url': "newUrl",
+      'github': "newGithub"
+    }
+
+  # TODO: update object tests most likely will be changed to use POST instead.
+  def test_update_author(self):
+    response = client.put(
+      f'/authors/{self.john._id}/',
+      data=json.dumps(self.payload),
+      content_type='application/json'
+    )
+    self.assertEqual(response.status_code, status.HTTP_200_OK)
+
```

Comment:
Should you get the author here and assert if its actually updated?

Reply:
Done.

Reply:
Now checking that the PUT'ed new fields are updated, and the rest are unchanged.